src/chart/pie/PieSeries.ts

```diff
@@ -107,6 +109,7 @@ export interface PieSeriesOption extends
     type?: 'pie'
 
     roseType?: 'radius' | 'area'
+    itemRadiusScale?: (dataParams: PieCallbackDataParams) => number
```

Comment:
I would suggest considering another name than `radius` because it may cause confusion with [emphasis.scaleSize](https://echarts.apache.org/en/option.html#series-pie.emphasis.scaleSize), which is the enlarging animation when emphasizing. You are welcomed to suggest a new name.

Reply:
`scalePercent`
`itemStyle.radius`
`itemStyle.scalePercent`

Reply:
I would personnally perfer `itemStyle.radiusPercent` to avoid using `scaleXXX`, but not with very strong preference. If only `itemStyle.xxx` is used but no `series.xxx`, I think we should skip the callback function.

In ECharts, for options support both value and percentage form (like [radius](https://echarts.apache.org/zh/option.html#series-pie.radius), we usually use strings like `'30%'` to represent percentage. But since we don't support number type in our case, I think a 0-1 number type is acceptable.